css/style.css

```diff
@@ -0,0 +1,508 @@
+.container {
+  background: url(../img/Background.png) no-repeat;
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  min-height: 1062px;
+  margin: auto;
+}
+
+.menu { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5821092)